keyValues2Normalized.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/python
```

Comment:
this code looks not used, i understand more or less the purpose, but i think it would be better in a separate PR

Reply:
also probably the simplest trick is the following:
load the keyValue example in contextBroker, query the model not using the keyValue, store the normalised example. Feelings?

Reply:
The code has been used to generate the normalized examples, and then some manual tweaking. Using a Context Broker just for that purpose I think could be overkilling.

Reply:
we use it anyhow for the validation, it would have been just an additional step ;)